src/frontend/App/typer/behandlingstype.ts

```diff
@@ -1,23 +1,20 @@
 export enum Behandlingstype {
     FØRSTEGANGSBEHANDLING = 'FØRSTEGANGSBEHANDLING',
     REVURDERING = 'REVURDERING',
-    BLANKETT = 'BLANKETT',
```

Comment:
Henter vi fortsatt ikke blanketter fra backend? Sånn att vi burde vel kanskje ikke fjerne dette før vi fjernet de fra backend?

Reply:
Begge deler skal fjernes, men vi tenkte fornt end først, for å hindre at noen oppretter nye.

Reply:
Man kan ikke opprette blanketter i frontend
Dette er kun visning av tidligere blanketter

Reply:
Det enkleste er å bare fjerne koden i backend som oppretter blankett, då den ikke burde brukes. Så kan man fjerne data i databasen, og sen kode koblet til blankett

Reply:
Det er ikke noen som opprettet blankett siden februar, så tror det skal gå fint å merge denne når vi senere slettet data i backend :)